src/ng/templateRequest.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var $compileMinErr = minErr('$compile');
+var $templateRequestMinErr = minErr('$compile');
```

Comment:
I figured this file shouldn't depend on compile.js like that. Might want to also remove the `'$compile'` string but I figured that would be another change...